src/containers/TemplateBuilder/shared/OperationContextHelpers.tsx

```diff
@@ -88,15 +95,18 @@ const checkMutationResult = async (
 
 export const updateTemplate: UpdateTemplateHelper =
   (setErrorAndLoadingState: SetErrorAndLoadingState, updateTemplateMutation) =>
-  async (id, patch) => {
+  async (template, patch) => {
```

Comment:
This wasn't strictly necessary to change. Originally, I was going to add the check for "can this be modified?" in here, and that would have required the versionId, etc of the template to determine. But then I found it was easier to just set the UI elements to "disabled" or hidden, so didn't need this. 

However, I've left it in as it's arguably no worse and it could come in handy in the future.